TestTables/CMIP6_Emon.json

```diff
@@ -1186,7 +1186,7 @@
         }, 
         "dissi14c": {
             "frequency": "mon", 
-            "modeling_realm": "ocnBgChem", 
+            "modeling_realm": "ocnbgchem", 
```

Comment:
@doutriaux1 eek, nope this should be `ocnBgchem` - so the only upper case is `B`

Reply:
https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_realm.json#L9

@taylor13

Reply:
@durack1 @taylor13 I thought it was supposed to be all lower case. My bad, will push an update right away